DailyHealth/CustomerLoginDisplay.java

```diff
@@ -0,0 +1,219 @@
+		loginPanel.setLayout(null);
+		
+		JLabel loginLabel = new JLabel("�α���", JLabel.CENTER);
+		loginPanel.add(loginLabel);
+		loginLabel.setFont(new Font("��� ���", Font.BOLD, 32));
+		loginLabel.setBounds(350, 90, 300, 60);
+		
+		// ���̵� ��
+		JLabel idLabel = new JLabel("���̵�");
+		loginPanel.add(idLabel);
+		idLabel.setBounds(290, 200, 70, 60);
+		idLabel.setFont(new Font("��� ���", Font.PLAIN, 16));
+		
+		// ���̵� textfield
+		idField = new JTextField();
+		loginPanel.add(idField);
+		idField.setBounds(390, 200, 300, 60);
+		idField.setFont(new Font("��� ���", Font.PLAIN, 24));
+		
+		
+		// ��й�ȣ textfield
+		JLabel pwLabel = new JLabel("��й�ȣ");
+		loginPanel.add(pwLabel);
+		pwLabel.setBounds(290, 285, 70, 60);
+		pwLabel.setFont(new Font("��� ���", Font.PLAIN, 16));
+		
+		// ��й�ȣ ��
+		pwField = new JPasswordField();
+		loginPanel.add(pwField);
+		pwField.setBounds(390, 285, 300, 60);
+		pwField.setFont(new Font("��� ���", Font.PLAIN, 24));
+		
+		// �α��� Ȯ�� ��ư
+		JButton loginBtn = new JButton("�α���");
+		loginPanel.add(loginBtn);
+		loginBtn.setBounds(360, 380, 130, 70);
+		loginBtn.setBackground(new Color(240, 237, 240));
+		loginBtn.setFont(new Font("��� ���", Font.PLAIN, 24));
+		
+		// �α��� Ȯ�� ��ư Ŭ�� �̺�Ʈ
+		loginBtn.addActionListener(new ActionListener() {
+			@Override
+			public void actionPerformed(ActionEvent e) {
+				if (CustomerLogin() == true) {
+					idField.setText("");
+					pwField.setText("");
+					frame.setVisible(false);
+					new CustomerDisplay();
+					CustomerDisplay.nameLabel.setText(name);
+                	CustomerDisplay.idLabel.setText(id);
+                	CustomerDisplay.addressLabel.setText(address);
+                	CustomerDisplay.phoneLabel.setText(phone);
+                	CustomerDisplay.ageLabel.setText(age);
+                	CustomerDisplay.currentWeightLabel.setText(current_weight);
+                	CustomerDisplay.targetWeightLabel.setText(target_weight);
+                	CustomerDisplay.heightLabel.setText(height);
+                	CustomerDisplay.remainDaysLabel.setText(remain_days);
+                	CustomerDisplay.bmiLabel.setText(bmi);
+                	CustomerDisplay.bmrLabel.setText(bmr);
+				}else {
+					idField.setText("");
+					pwField.setText("");
+				}
+			}
+		});
+		
+		// �α��� ��� ��ư
+		JButton loginExitBtn = new JButton("���");
+		loginPanel.add(loginExitBtn);
+		loginExitBtn.setBounds(520, 380, 130, 70);
+		loginExitBtn.setBackground(new Color(218, 226, 234));
+		loginExitBtn.setFont(new Font("��� ���", Font.PLAIN, 24));
+		
+		// �α��� ��� ��ư Ŭ�� �̺�Ʈ
+		loginExitBtn.addActionListener(new ActionListener() {
+			@Override
+			public void actionPerformed(ActionEvent e) {
+				frame.setVisible(false);
+				new LogoChoice();
+			}
+		});
+		
+	}//end of MainDisplay(and ����)
+		
+	public static void main(String[] args) {
+		new CustomerLoginDisplay(); //Member-List Ŭ���� ����
+	}//end of main
+
+	public boolean CustomerLogin() {

```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.